docs/guides/software-lifecycle/continuous-testing/starter-kit/integration-test-pytest.md

```diff
@@ -0,0 +1,53 @@
+# Integration test using Pytest
+
+## Prerequisites
+```
+pip install pytest
+```
+
+
+## Example
```

Comment:
What can we recommend to actually simplify the generation of unit tests? Consider researching some tools to generate test templates or something more advanced, such as [hypothesis](https://hypothesis.readthedocs.io/en/latest/quickstart.html). These starter kits would be much more useful if unit tests could be generated from existing code a reader might already have. Automation is definitely better than asking someone to write tests by hand in this context.